### ISSUE TITLE
vmagent: Reload on remote relabel config change

### ISSUE THREAD

**Issue by jsanant** [VictoriaMetrics/VictoriaMetrics#518]
**Is your feature request related to a problem? Please describe.**
When a new metric to be dropped is added to the config file for `-remoteWrite.relabelConfig` or `-remoteWrite.urlRelabelConfig`, a restart of vmagent is required, this can cause downtime when applications are sending metrics to vmagent. 

**Describe the solution you'd like**
A hot reload  via HTTP handler or via SIGHUP signal, similar to the reload for `-promscrape.config` would help out. 

**Describe alternatives you've considered**
As of now, restarting the vmagent solves the issue.

**Comment by valyala** [comment]
`vmagent` should start reloading `-remoteWrite.relabelConfig` and `-remoteWrite.urlRelabelConfig` after the commit 32652485e3580e54e9fa3e386988e9a2186d7c43 .

@jsanant , could you build `vmagent` from this commit according to [these docs](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmagent/README.md#how-to-build-from-sources) and verify whether it works as expected?

**Comment by jsanant** [comment]
@valyala - It's working as expected. Thank you for the fast turn around time on this feature request! 

Here are the logs when I send a SIGHUP signal to the vmagent process: 

```
{"ts":"2020-05-30T16:37:55.626Z","level":"info","caller":"lib/promscrape/scraper.go:106","msg":"SIGHUP received; reloading Prometheus configs from \"/etc/prometheus/prometheus.yml\""}
{"ts":"2020-05-30T16:37:55.626Z","level":"info","caller":"app/vmagent/remotewrite/remotewrite.go:91","msg":"SIGHUP received; reloading relabel configs pointed by -remoteWrite.relabelConfig and -remoteWrite.urlRelabelConfig"}
{"ts":"2020-05-30T16:37:55.626Z","level":"info","caller":"lib/promscrape/scraper.go:113","msg":"nothing changed in \"/etc/prometheus/prometheus.yml\""}
{"ts":"2020-05-30T16:37:55.627Z","level":"info","caller":"app/vmagent/remotewrite/remotewrite.go:98","msg":"Successfully reloaded relabel configs"}
```

**Comment by valyala** [comment]
@jsanant , thanks for the confirmation!

This feature has been included in [v1.36.3](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing it as done.
